include/States.h

```diff
@@ -79,7 +80,8 @@ class States {
    * Perform a cache reduction. After running this, getStates(), getPositions(),
    * and getVelocities() will return vectors of reduced size, and getState(),
    * getPosition(), and getVelocity() will
-   * returns values interpolated over the reduced cache using a cubic hermite spline
+   * returns values interpolated over the reduced cache using a cubic hermite splineGSW#|
+   * 1111111
```

Comment:
Looks like something hit the keyboard here.

Reply:
Really not sure how these mistakes happened, I can't blame a cat I don't have 🤔

Reply:
:cat: